### DISCUSSION TITLE
Account Activation Pending Contact Admin for WebUI Access

### DISCUSSION THREAD

**Discussion by blue-eyed-g** [open-webui/open-webui#1566] [General]
I logged in the UI interface for the first time and it showed 
"Account Activation Pending Contact Admin for WebUI Access
Your account status is currently pending activation. To access the WebUI, please reach out to the administrator. Admins can manage user statuses from the Admin Panel." 
after I registered the account. How can I enter the Administrator mode?

**Comment by ghost** [comment]
Read the [FAQ](https://docs.openwebui.com/faq#q-i-tried-to-login-and-couldnt-made-a-new-account-and-now-im-being-told-my-account-needs-to-be-activated-by-an-admin). Here are steps to [manually reset the admin password](https://docs.openwebui.com/getting-started/troubleshooting#reset-admin-password) if you're unwilling to wipe the data and begin fresh. You don't mention your installation method nor access method (browser? make and version?), but using the recommended official container I am unable to re-produce the described problem.

**Reply by lonngxiang** [reply]
Page Not Found

**Reply by justinh-rahb** [reply]
https://docs.openwebui.com/troubleshooting/#reset-admin-password

**Reply by laowalter** [reply]
I tried, so frustrated about https://docs.openwebui.com/troubleshooting/#reset-admin-password,

JUST CLEAR THE VOLUME.

```
sqlite3 backend/data/webui.db "UPDATE auth SET password='$2y$10$COK0U5fOIB5yqA/qeTVxk.iJRWUa1MNCbQCnVb6gDoYFDRhykxn8O' WHERE email='c@c.com';"
Error: unable to open database "backend/data/webui.db": unable to open database file
```

and

```
docker run --rm -v open-webui:/data alpine/socat EXEC:"bash -c 'apk add sqlite && echo UPDATE auth SET password='\''$2y$10$COK0U5fOIB5yqA/qeTVxk.iJRWUa1MNCbQCnVb6gDoYFDRhykxn8O'\'' WHERE email='\''c@c.com'\''; | sqlite3 /data/webui.db'", STDIO
2024/07/06 01:32:10 socat[7] E execvp("bash", "bash", "-c", "apk", ...): No such file or directory
2024/07/06 01:32:10 socat[1] W waitpid(): child 7 exited with status 1
```

**Reply by poing** [reply]
#### Alternate Docker Method

_If you have issues with the password reset._  I had issues chaining the `bash` commands in `alpine/socat`.

1. **Run `alpine` linux connected to the open-webui volume.**

    ```bash
    docker run -it --rm -v open-webui:/path/to/data alpine
    ```
    _`/path/to/data` depends on __your__ volume settings._
    
    1. Install `apache2-utils` and `sqlite`:
    
        ```sh
        apk add apache2-utils sqlite
        ```
    1. Generate `bcrypt` hash:
    
        ```sh
        htpasswd -bnBC 10 "" your-new-password | tr -d ':'
        ```
        
    1. Update password:
    
        ```sh
        sqlite3 /path/to/data/webui.db
        ```
        
        ```sql
        UPDATE auth SET password='HASH' WHERE email='admin@example.com';
        -- exit sqlite: [Ctrl + d]

**Comment by blue-eyed-g** [comment]
thanks, I installed this before, and it kept all the original data. I just cleared the original volume in docker

**Comment by ViesLink** [comment]
Written to others who have similar issues

1. If this prompt appears for the first login, and you are an administrator (the instance is running on your own computer or server), it is likely that it is a residual problem of docker images, which can be fixed by cleaning up Docker Volume. 

2. If you are logging in to the services provided by other individuals or organizations, you need to change the account status in the administrator panel, and after the change, refresh the page that prompts you can log in normally.

**Comment by Chaithra-lm** [comment]
even if I clear whole volume and reset password still it says Account Activation Pending Contact Admin for WebUI Access and I uninstalled and installed docker itself. still .... not resolve can anybody help me I am stuck

**Comment by jzhang82119** [comment]
I am having the same issue. I tried all the method mentioned in this thread. Wonder where the admin password stored. What is the best clean way to start over?

**Comment by soldcoin** [comment]
This works for me. (Note: one `\` for `$`, eg: \\$2y\\$10\\$ZlsB0EmkjJ0PVaIr2bc0BOp4g5/fGLeZQqalwRwYXtY9xixtfBlEW)
```bash
docker run --rm -v open-webui:/data alpine sh -c 'apk add --no-cache sqlite && sqlite3 /data/webui.db "UPDATE auth SET password='\''HASH'\'' WHERE email='\''admin@example.com'\'';"'
```

**Comment by nPrevail** [comment]
How does one solve this if they use NixOS? I don't use docker....
